### ISSUE TITLE
join

### ISSUE THREAD

**Issue by Genius398** [Coders-Collab/Coders-Collab#1]
@somePythonProgrammer and @Super45Coder join here

**Comment by somePythonProgrammer** [comment]
Ok

**Comment by somePythonProgrammer** [comment]
Also, what is this organization doing right now?
I can help!

**Comment by Genius398** [comment]
> Also, what is this organization doing right now?
> I can help!

yes! Im working on the main website. I have no idea about the other stuff tho. BTW I'll invite you to my fork then we can work on it

**Comment by Super45coder** [comment]
Sure. I am in
________________________________

**Comment by Adatta1276** [comment]
Hi !!

**Comment by Adatta1276** [comment]
This is just a organization I made, we're working on the main website right now, and then we can try our hand at making some apps and games! I'm also working on a 3d game right now, which we'll publish once it's done! Link : https://github.com/Adatta1276/SpeedAndPolice , join the discord and go to the '3d-development' channel!

**Comment by somePythonProgrammer** [comment]
Sure!
